packages/webcert/src/feature/certificate/Modals/WarningNormalOriginModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import { CustomButton, InfoBox, ResourceLinkType, ModalBase } from '@frontend/common/src'
+import { useSelector } from 'react-redux'
+import { getUser, getUserResourceLink } from '../../../store/user/userSelectors'
+
+const WarningNormalOriginModal: React.FC = () => {
+  const user = useSelector(getUser)
+  const resourceLink = useSelector(getUserResourceLink(ResourceLinkType.WARNING_NORMAL_ORIGIN))
+  const [open, setOpen] = useState(true)
+
+  const handleClose = () => {
+    if (!user) {
+      return
+    }
+    setOpen(false)
+    window.sessionStorage[user.loggedInUnit.unitId] = true
```

Comment:
I'm not so used to using sessionStorage so I'm just curious why we are using it here?

Reply:
It was used in the original client to make sure that the modal only opens once per session.